packages/core/src/styles/typography/_typography.scss

```diff
@@ -17,7 +17,6 @@
   font-family: $cds-token-typography-font-family;
   margin-top: 0;
   margin-bottom: 0;
-  width: 100%;
```

Comment:
This style was breaking centering in vertical layouts and overrides user defined styles unnecessarily.